source/administration/changelog.md

```diff
@@ -2,6 +2,148 @@
+
+### Bug Fixes
+- Fixed line wrapping of the timestamp in Account Settings > Security > View Access History 
+- Fixed an inconsistent error message when creating a channel with a display name of two or less characters
+- Removed the duplicate "Back" button on the Team Creation page
+- The AltGR key no longer triggers keyboard shortcuts
+- Saving a team name without making changes no longer throws an error message
+- Group messages are now sorted alphabetically with direct messages
+- The "Create Channel" button will no only appear in the "More Channels" modal when the user has the permission to create channels
+- The Town Square channel menu no longer has redundant dividers with certain combinations of System Console > Policy settings
+- Fixed an issue where some conversations would not trigger the channel to appear unread in the left-hand sidebar
+- Removed emoji reactions no longer appear as a blank space
+- Fixed an issue where usernames sometimes did not appear when hovering over reactions
+- Fixed missing search results when using the "in:" or "from:" modifiers
+- Editing a custom theme no longer causes the user interface to freeze
+- The date of the root post in the right-hand sidebar now accurately displays the date
+- Fixed an issue where link previews would sometimes cause a horizontal scroll bar to appear
+- iOS code blocks no longer wrap to the next line
+
+### Compatibility  
+
+Changes from v3.7 to v3.8:
+- The **System Console > Configuration > [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url)** field is now mandatory. Please set the Site URL in the System Console or in the `gitlab.rb` file if you are using GitLab Mattermost.
+- Server logs will now be written to the `mattermost.log` file located at the path specified in **System Console > Logging > [File Log Directory](https://docs.mattermost.com/administration/config-settings.html#file-log-directory)**. Please set the file path in the System Console or in the `gitlab.rb` file if you are using GitLab Mattermost.
+- Backwards compatibility with the old CLI tool is removed in v3.8. See [documentation to learn more about the new CLI tool](https://docs.mattermost.com/administration/command-line-tools.html).
+
+#### config.json   
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json` or the System Console.
+
+**Changes to Team Edition and Enterprise Edition**:
+
+ - Under `EmailSettings` in `config.json`:
+   - Added `"SkipServerCertificateVerification": false` to skip verification of smtp server certificates.
+
+**Additional Changes to Enterprise Edition**:
+
+ - Under `TeamSettings` in `config.json`:
+   - Added `"RestrictPrivateChannelManageMembers": all` to set who can add and remove members from private groups.
+
+### Database Changes from v3.7 to v3.8
+
+XXXXXX check with dev
```

Comment:
I believe the only change was a new column `IsPinned` in the `Posts` table: https://github.com/mattermost/platform/commit/3ada7a41a7fb13abef19dd63dc56b720900dbaa9#diff-a788391365ece46cdf5c98badcdc3c32R251